File: back/src/main/kotlin/br/ufsc/mariaeduardamelohang/servidormatriculaufscpwa/model/database/Horario.java
package br.ufsc.mariaeduardamelohang.servidormatriculaufscpwa.model.database;

import static javax.persistence.GenerationType.IDENTITY;

import java.io.Serializable;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;
import javax.persistence.Table;

import lombok.Getter;

@Entity
@Table(name = "TB_HORARIO")
@Getter
public class Horario implements Serializable {

	private static final long serialVersionUID = 5040258127046402484L;

	@Id
	@GeneratedValue(strategy = IDENTITY)
	private Long id;

	@Column(nullable = false)
	private String horario;

	@Column(nullable = false)
	private boolean isUltimoHorarioPeriodo = false;
}

File: back/src/main/kotlin/br/ufsc/mariaeduardamelohang/servidormatriculaufscpwa/model/Horario.kt
package br.ufsc.mariaeduardamelohang.servidormatriculaufscpwa.model

import java.time.LocalTime
import javax.persistence.Column
import javax.persistence.Entity
import javax.persistence.GeneratedValue
import javax.persistence.GenerationType.IDENTITY
import javax.persistence.Id
import javax.persistence.Table

@Entity
@Table(name = "TB_HORARIO")
class Horario(
    @Id
    @GeneratedValue(strategy = IDENTITY)
    var id: Long? = null,

    @Column(nullable = false)
    val horario: LocalTime
)